src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -60,7 +60,16 @@ static int Last2DigitsTo0(int versionBuild)
                 _knownManifestIds = File.ReadAllLines(knownManifestIdsFilePath).Where(l => !string.IsNullOrEmpty(l)).ToHashSet();
             }
 
-            var manifestDirectory = Path.Combine(_sdkRootPath, "sdk-manifests", _sdkVersionBand);
+            string? userManifestsDir = userProfileDir is null ? null : Path.Combine(userProfileDir, "sdk-manifests", _sdkVersionBand);
+            string dotnetManifestDir = Path.Combine(_sdkRootPath, "sdk-manifests", _sdkVersionBand);
+            if (userManifestsDir != null && WorkloadInstall.IsUserLocal(_sdkRootPath, sdkVersion) && Directory.Exists(userManifestsDir))
+            {
+                _manifestDirectories = new[] { userManifestsDir, dotnetManifestDir };
```

Comment:
So we still use any manifests in the dotnet dir even when we're running user local? Is the intention that these baseline manifests are only used at first and once we get manifest updates they're installed to the user local folder? Then, at that point, we shouldn't be looking at the dotnet baseline manifests at all, right?

Reply:
Yes, on a clean install it should use the manifests from the dotnet directory.  The first time we update the manifests they will be saved in the user folder, and it will stop using the ones in the dotnet directory.